.github/workflows/release-cloud.yml

```diff
@@ -5,78 +5,38 @@ on:
-          - alpha
-          - patch
-          - minor
-          - major
-          - release-candidate
+        description: "Release type. One of alpha, patch, minor, major, release-candidate"
+        type: string
       release-version:
+        description: "Set an explicit version, that will overwrite release-type. Fails if version is not compliant."
         type: string
-        description: Set an explicit version, that will overwrite release-type. Fails if version is not compliant.
-    secrets: inherit
+    secrets:
+      GREENBONE_BOT:
+        required: true
+      GREENBONE_BOT_MAIL:
+        required: true
+      GREENBONE_BOT_TOKEN:
+        required: true
+      GPG_KEY:
+        required: false
+      GPG_FINGERPRINT:
+        required: false
+      GPG_PASSPHRASE:
+        required: false
 
 jobs:
   build-and-release:
-    name: Create a new release with pontos
-    # If the event is a workflow_dispatch or on of the labels 'pre release',
-    # 'patch release', 'minor release' or 'major release' is set and PR is
-    # closed because of a merge
-    # NOTE: priority of set labes will be alpha > release-candidate > patch > minor > major,
-    #       so if 'major' and 'patch' labes are set, it will create a patch release.
-    if: |
-      ( github.event_name == 'workflow_dispatch') || (
-        ( contains(github.event.pull_request.labels.*.name, 'alpha release') ||
-          contains(github.event.pull_request.labels.*.name, 'rc release') ||
-          contains(github.event.pull_request.labels.*.name, 'patch release') ||
-          contains(github.event.pull_request.labels.*.name, 'minor release') ||
-          contains(github.event.pull_request.labels.*.name, 'major release')) &&
-          github.event.pull_request.merged == true )
-    runs-on: 'ubuntu-latest'
-    steps:
-      - name: Selecting the Release type
-        if: contains(github.event.pull_request.labels.*.name, 'major release')
-        run: |
-          echo "RELEASE_TYPE=major" >> $GITHUB_ENV
-      - if: contains(github.event.pull_request.labels.*.name, 'minor release')
-        run: |
-          echo "RELEASE_TYPE=minor" >> $GITHUB_ENV
-      - if: contains(github.event.pull_request.labels.*.name, 'patch release')
-        run: |
-          echo "RELEASE_TYPE=patch" >> $GITHUB_ENV
-      - if: contains(github.event.pull_request.labels.*.name, 'rc release')
-        run: |
-          echo "RELEASE_TYPE=release-candidate" >> $GITHUB_ENV
-      - if: contains(github.event.pull_request.labels.*.name, 'alpha release')
-        run: |
-          echo "RELEASE_TYPE=alpha" >> $GITHUB_ENV
-      - name: Workflow_dispatch RELEASE_TYPE 
-        if: github.event_name == 'workflow_dispatch'
-        run: | 
-          echo "RELEASE_TYPE=${{ github.event.inputs.release-type }}" >> $GITHUB_ENV
-
-      - name: Echoing the release type
-        run: |
-          echo $RELEASE_TYPE
+    name: Create a new release
+    uses: ./.github/workflows/release-3rd-gen.yml
```

Comment:
Can we create a `release.yml` and call that workflow from both specific release workflows?

Reply:
yes we can